src/web/app/pages-instructor/instructor-search-page/comment-result-table/comment-result-table.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="card border border-primary mb-4">
+  <div class="card-header text-white bg-primary font-weight-bold">Questions, responses, comments on responses</div>
+  <div class="card-body p-4 row" *ngFor="let sessions of commentTables">
+    <div class="font-weight-bold col-md-3">Session: {{sessions.sessionName}} {{sessions.courseId}}</div>
+    <div class="col-md-9">
+      <div *ngFor="let section of sessions.sections" class="card p-0 border border-primary mb-5">
+        <div class="card-header bg-light-blue">
+          <b>Question {{section.questionNumber}}</b> {{section.questionText}}
+        </div>
+        <div *ngFor="let responseComment of section.responseComments">
```

Comment:
Avoid naming like responseComments? If you remember the DB entity relation, the name refers to a different thing.